posthog/warehouse/models/external_data_schema.py

```diff
@@ -147,6 +147,14 @@ def partition_mode(self) -> PartitionMode | None:
 
         return None
 
+    @property
+    def partition_format(self) -> PartitionFormat | None:
+        # This key doesn't get reset on pipeline_reset and can only be set via the DB directly right now
```

Comment:
This could be in the method's docstring, so that IDEs, `help`, and other tools will make it more visible.